python/packages/sdk-schema/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### 0.2.0 (2023-03-28)
+
+### ⚠ BREAKING CHANGES
+
+- Switch to protoc compiler instead of betterproto
```

Comment:
Why exactly it's a breaking change? Does it change the way in how this dependency is used once installed?

Reply:
It's a breaking change, because the generated classes are different, they have different attributes and methods etc. It also does change how the dependency is used in terms of import path.